Podfile

```diff
@@ -3,7 +3,7 @@ source 'https://github.com/CocoaPods/Specs.git'
 use_frameworks!
 
 pod 'IQKeyboardManagerSwift', '~>4.0'
-pod 'Babbage', podspec: 'AppConnectSwift/Babbage.podspec'
+pod 'Babbage', podspec: 'AppConnectSwift/Babbage.podspec.json'
```

Comment:
I think we should still use a `.podspec` file rather than a `podspec.json`. It's more standard and is what we use elsewhere.